splunk_add_on_ucc_framework/splunktaucclib/common/log.py

```diff
@@ -0,0 +1,49 @@
+import sys
+import logging
+import splunktalib.common.log as stclog
```

Comment:
does this need to be changed?

Reply:
@rfaircloth-splunk 
Splunktalib is used in mscs and it is statically commited in mscs repo
So should we also move the splunktalib to ucc repo?